public/client.js

```diff
@@ -165,9 +167,8 @@ socket.on('Display Message', function(data) {
     
     let currRoom = $(".active").attr("id");
     let isJoined = $("#" + room_id + "-msg").attr("data-joined");
-
     if (socket.username != data.user && currRoom != data.room && isJoined == 1) {
-        $.notify(`Room ${data.room}-\n${data.user}: ${(p.length >= 20) ? p.substr(0, 20) + '...' : p}`, "info");
+        $.notify(`${data.user}@${data.room}\n${(p_notif.length >= 20) ? p_notif.substr(0, 20) + '...' : p_notif}`,"info");
```

Comment:
Can you keep notification format as it was?
I think replacing p with p_notif should be sufficient.